tailwind.config.js

```diff
@@ -332,10 +332,20 @@ module.exports = {
           from: { opacity: 0 },
           to: { opacity: 0.1 },
         },
+        scaleIn: {
+          from: { opacity: "0", transform: "rotateX(-10deg) scale(0.9)" },
+          to: { opacity: "1", transform: "rotateX(0deg) scale(1)" },
+        },
+        scaleOut: {
+          from: { opacity: "1", transform: "rotateX(0deg) scale(1)" },
+          to: { opacity: "0", transform: "rotateX(-10deg) scale(0.95)" },
+        },
       },
       animation: {
         "accordion-down": "accordion-down 0.2s ease-out",
         "accordion-up": "accordion-up 0.2s ease-out",
+        scaleIn: "scaleIn 200ms ease",
```

Comment:
Can we match the kebab-case convention of the other animations here to be more consistent?